corehq/toggles.py

```diff
@@ -106,8 +106,12 @@ def __init__(self, slug, label, tag, namespaces=None, help_link=None,
         # updated.  This is only applicable to domain toggles.  It must accept
         # two parameters, `domain_name` and `toggle_is_enabled`
         self.save_fn = save_fn
-        self.always_enabled = always_enabled or set()
-        self.always_disabled = always_disabled or set()
+        # Toggles can be delcared in localsettings statically
+        #   to avoid cache lookups
+        self.always_enabled = (always_enabled or set() |
+            set(settings.STATIC_TOGGLE_STATES.get(self.slug, {}).get('always_enabled', [])))
+        self.always_disabled = (always_disabled or set() |
```

Comment:
nit: why combine with empty set?
```suggestion
        self.always_disabled = (always_disabled or
```
Also, `or` operator should be moved to the line below to avoid stickler comment.

The same applies two lines above.

Reply:
I'm guessing the intention here was `(always_disabled or set()) | set(...))` and not `always_disabled or (set() | set(...))`. The code does the latter (I had to check in my python shell to be sure!), but was maybe intended to do the former.

Reply:
Thanks for the catch, I was aware of this but I must have fumbled up the braces latter and missed it.

Reply:
https://github.com/dimagi/commcare-hq/pull/26120